avm/res/machine-learning-services/workspace/main.bicep

```diff
@@ -547,6 +562,87 @@ type featureStoreSettingType = {
   onlineStoreConnectionName: string?
 }?
 
+@discriminator('type')
+type OutboundRuleType = FqdnOutboundRuleType | PrivateEndpointOutboundRule | ServiceTagOutboundRule
+
+type FqdnOutboundRuleType = {
+  @sys.description('Required. Type of a managed network Outbound Rule of a machine learning workspace. Only supported when \'isolationMode\' is \'AllowOnlyApprovedOutbound\'.')
```

Comment:
I guess it could also be 'Optional' if it only allows a single value?

Reply:
The `type` parameter is required for the `OutboundRuleType` type, it has to be `FQDN`, `PrivateEndpoint` or `ServiceTag`

It just looks like it only has one value here because we're using `@discriminator('type')`

Reply:
Ah, dang - your tricked me with Bicep's syntactic sugar. Well played @cecheta, well indeed